### ISSUE TITLE
link to Org guidance 

### ISSUE THREAD

**Issue by stevieflow** [IATI/IATI-Schemas#97]
These schema descriptions need to link to Org ID guidance:

`iati-identifier`:
- [ ] http://dev.iatistandard.org/organisation-standard/iati-organisations/iati-organisation/iati-identifier/
- [ ] http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/iati-identifier/

`reporting-org`:
- [ ] http://dev.iatistandard.org/organisation-standard/iati-organisations/iati-organisation/reporting-org/
- [ ] http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/reporting-org/

`recipient-org`:
- [ ] http://dev.iatistandard.org/organisation-standard/iati-organisations/iati-organisation/recipient-org-budget/recipient-org/

`participating-org`: 
- [ ] http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/participating-org/

In ''transaction``:
- [ ] `provider-org`: http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/transaction/provider-org/
- [ ] `receiver-org`: http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/transaction/receiver-org/

`other-identifier`:
- [ ] http://dev.iatistandard.org/activities-standard/iati-activities/iati-activity/other-identifier/

**Comment by caprenter** [comment]
I think we need to think about this. 

One option might be to include the link in Extra Documentation rather than in the schema definition.

There is talk about removing some URLs from the schema.

I don't think this is necessary for the 1.04 milestone, but should be resolved soon afterwards, so bumping to After 1.04 milestone.

**Comment by bill-anderson** [comment]
I would stick to validation / rule set descriptions in the schema and link to Extra Documentation for a full explanation and guidance

**Comment by stevieflow** [comment]
@Bjwebb in 1.04 - did this get resolved via all URLs for this being changed to  http://iatistandard.org/org-ref ?

**Comment by Bjwebb** [comment]
No, I only _removed_ the URLs suggesting codes should be on the organisation identifiers "codelist"
